### ISSUE TITLE
Add option to replicate pager on issue page also on the top

### ISSUE THREAD

**Issue by tterranigma** [sindresorhus/refined-github#1541]
You now have to scroll all the time. With a 130% page zoom, I get around 8 issues per screen. I have to scroll a bit to get to the pager and it gets tiring if I want to visit every page and do Ctrl-F to search for a specific word in the issue titles.

**Comment by sindresorhus** [comment]
I don't think that would look good. Would be better to add keyboard shortcuts for going to next/previous issue page.

**Comment by sindresorhus** [comment]
> I have to scroll a bit to get to the pager and it gets tiring if I want to visit every page and do Ctrl-F to search for a specific word in the issue titles.

Why are you not using the actual search? For example, `in:title foo`

**Comment by tterranigma** [comment]
I didn't know about `in:title`, thanks. Still, shortcuts would be really handy. How can I add them? Is there an option somewhere?

**Comment by sindresorhus** [comment]
It has to be added to Refined GitHub as a feature, ala https://github.com/sindresorhus/refined-github/blob/master/source/features/add-create-release-shortcut.js

**Comment by rzpio** [comment]
I'm on it if it's okay, @tterranigma.

**Comment by bfred-it** [comment]
There’s also “cmd+down” (OS X) or “end” key (Windows) to scroll to the bottom of the page
